src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -406,6 +407,7 @@ public VoldemortConfig(Props props) {
 
         this.modifiedProtocol = props.getString("readonly.modify.protocol", null);
         this.modifiedPort = props.getInt("readonly.modify.port", -1);
+        this.bouncyCastleEnabled = props.getBoolean("readonly.bouncy.castle.enabled", false);
```

Comment:
I would call this parameter,  use.bouncycastle.for.ssl and initialize this here itself.  As the way it is done in HdfsFetcher it impacts all the SSL from that point on, so it would be safer to do it this way I believe.